Makefile

```diff
@@ -140,6 +141,11 @@ ol: OL_JS = ol.js ol-debug.js
 ol: scripts/ol-geoadmin.json .build-artefacts/ol3 .build-artefacts/ol-requirements-installation.timestamp
 	cd .build-artefacts/ol3; \
 	git reset HEAD --hard; \
+	ifdef ${OL3_VERSION} \
+	git checkout ${OL3_VERSION}; \
+	else \
+	git checkout $(OL3_VERSION); \
+	enif; \
```

Comment:
You could use `OL3_VERSION ?= tags/v3.4.0` in the definition part. It will set OL3_VERSION varaible to OL3_VERSION parameter when given, otherwise to tags/v3.4.0 (as is already done for the DEPLOY_TARGET variable)